vendor/bat-native-ads/src/bat/ads/history_item_value_util.cc

```diff
@@ -54,21 +54,23 @@ base::Value::List HistoryItemToDetailRowsValue(
 HistoryItemInfo HistoryItemFromValue(const base::Value::Dict& root) {
   HistoryItemInfo history_item;
 
-  if (const auto* value = root.FindString(kCreatedAtKey)) {
+  if (const auto* created_at_value_string = root.FindString(kCreatedAtKey)) {
```

Comment:
Are these changes needed?

Reply:
Fixed in b4174716b278ed73d7aab2c1ed841d54e25cb51a.